settings/NoticePolicy/NoticePolicySettings.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { sortBy } from 'lodash';
+import { FormattedMessage } from 'react-intl';
+
+import { EntryManager } from '@folio/stripes/smart-components';
+
+import NoticePolicyDetail from './NoticePolicyDetail';
+import NoticePolicyForm from './NoticePolicyForm';
+import validate from '../Validation/NoticePolicy';
+import NoticePolicy from '../Models/NoticePolicy';
+
+class NoticePolicySettings extends React.Component {
+  static manifest = Object.freeze({
+    noticePolicies: {
+      type: 'okapi',
+      records: 'noticePolicies',
+      path: 'notice-policy-storage/notice-policies',
+    },
+  });
+
+  static propTypes = {
+    resources: PropTypes.shape({
+      noticePolicies: PropTypes.object,
+    }).isRequired,
+    mutator: PropTypes.shape({
+      noticePolicies: PropTypes.shape({
+        POST: PropTypes.func,
```

Comment:
Please add isRequired to errors property or defaultProps if this property is not required
```suggestion
        POST: PropTypes.func.isRequired,
```